src/main/java/ee/tuleva/onboarding/mandate/application/ApplicationService.java

```diff
@@ -66,7 +68,7 @@ public boolean hasPendingWithdrawals(Person person) {
   }
 
   private List<TransferApplication> groupTransfers(List<ApplicationDTO> transferApplications) {
-    val locale = localeService.getCurrentLocale();
+    String language = localeService.getCurrentLocale().getLanguage();
```

Comment:
According to the [law of demeter](https://en.wikipedia.org/wiki/Law_of_Demeter#In_object-oriented_programming) we should ask for the language directly from the localeService. I.e. refactor it in a way that we would do: `localeService.getCurrentLanguage()` instead.

Reply:
Turns out there already was such a method. I'm a fool!